mlrun/feature_store/retrieval/job.py

```diff
@@ -79,9 +79,16 @@ def _is_ready(self):
             raise mlrun.errors.MLRunTaskNotReady("feature vector dataset is not ready")
         self.vector.reload()
 
-    def to_dataframe(self):
-        """return result as dataframe"""
-        return mlrun.get_dataitem(self.target_uri).as_df()
+    def to_dataframe(self, columns=None, df_module=None, **kwargs):
+        """return result as a dataframe object (generated from the dataitem).
+
+        :param columns:   optional, list of columns to select
+        :param df_module: optional, target dataframe class (e.g. pd, dd, cudf, ..)
```

Comment:
I think this message is a bit confusing. One might think it's a string.

Reply:
edited